README.md

```diff
@@ -122,7 +120,6 @@ See how much simpler that is? You won't need to add `done` to the `it` or accide
 [Dependencies]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master
 [dependencies-image]: https://david-dm.org/tests-always-included/jasmine-node-helpers/master.png
 [Jasmine]: https://jasmine.github.io/
-[Jasmine-Node]: https://www.npmjs.com/package/jasmine-node
```

Comment:
Added the line back in. 

Linking to the package rather than the documentation makes more sense. That link has been changed as well.